rest/src/main/kotlin/builder/message/MessageCreateBuilder.kt

```diff
@@ -82,6 +85,16 @@ class MessageCreateBuilder : RequestBuilder<MultipartMessageCreateRequest> {
         allowedMentions = (allowedMentions ?: AllowedMentionsBuilder()).apply(block)
     }
 
+    @OptIn(ExperimentalContracts::class)
+    @KordPreview
+    inline fun components(builder: CompositeActionRowBuilder.() -> Unit) {
+        contract {
+            callsInPlace(builder, InvocationKind.EXACTLY_ONCE)
+        }
+
+        components = CompositeComponentBuilder().apply(builder).components
+    }
```

Comment:
Kord's builders should generally be additive, previous values shouldn't be cleared.